src/containers/WarningConfirmThresholdScenario/index.jsx

```diff
@@ -49,28 +49,50 @@ class WarningConfirmThresholdScenario extends React.Component {
 		if (!isValid) {
 			return;
 		}
-		const nextTreshold = this.props.treshold.value;
-		let { permissionsKeys } = this.props;
-		const { network, account } = this.props;
-		const accountId = account.toJS().id;
-		permissionsKeys = permissionsKeys.toJS();
-		const activeKeysData = {
-			keys: permissionsKeys.active.keys.concat(permissionsKeys.active.accounts),
-			threshold: permissionsKeys.active.threshold,
-		};
-		const maxNextThreshold = activeKeysData.keys
-			.reduce((accumulator, currentKey) => accumulator + currentKey.weight, 0);
-		if (maxNextThreshold < nextTreshold) {
+		const permissionsKeys = this.props.form.toJS();
+		const network = this.props.network.toJS();
+		const accs = JSON.parse(localStorage.getItem(`accounts_${network.name}`));
+		const nextTreshold = new BN(this.props.treshold.value).toString(10);
+		const { keys, accounts } = permissionsKeys.active;
+		let enoughNextThreshold = new BN(0).toString(10);
+		let maxNextValue = new BN(0).toString(10);
```

Comment:
don't need to create and transform to string every time
actually transform this not necessary at all